### DISCUSSION TITLE
Инструкция: Как опубликовать свои обновления в репозиторий

### DISCUSSION THREAD

**Discussion by yuldashov10** [shox-py/.github#7] [Q&A] [answered]
1. Откройте Git Bash (Windows) или терминал (Mac OS, Linux) на Вашем компьютере.
2. Перейдите в папку, где расположен локальный репозиторий, который Вы хотите обновить. Для этого выполните команду `cd path/to/repository`, 
   * где `path/to/repository` - путь к вашему локальному репозиторию.
3. Убедитесь, что находитесь на нужной ветке репозитория. Чтобы узнать текущую ветку, выполните команду `git branch`. 
   * Ветка, в которой Вы находитесь, будет отмечена символом `*`.
4. Выполните команду `git status`, чтобы проверить, какие файлы были изменены или добавлены.
5. Добавьте измененные файлы в индекс с помощью команды `git add <file>`. 
    * Здесь `<file>` может быть путем к файлу или названием директории. 
    * Если Вы хотите добавить несколько файлов, перечислите их через пробел: `git add <file1> <file2> <file3>`. 
    * Если Вы хотите сразу добавить все измененные файлы, выполните команду `git add .` (обратите внимание на точку).
6. Затем выполните команду `git commit -m "Текст Вашего коммита"`, чтобы зафиксировать изменения. 
    * Вместо "Текст Вашего коммита" напишите краткое описание Ваших изменений.
7. После фиксации изменений выполните команду `git push origin <branch>`, чтобы опубликовать свои изменения на удаленный репозиторий GitHub.
    * Здесь `<branch>` - название ветки, в которую Вы хотите опубликовать изменения.
8. Если Ваш Git настроен для использования `HTTPS` протокола, Git Bash или терминал запросят Ваше имя пользователя и пароль от Вашей учетной записи GitHub.
9. Готово! После успешной передачи изменений на GitHub Ваш локальный репозиторий будет синхронизирован с удаленным репозиторием.

**Comment by abcsmeal-ai** [comment, marked as answer]
Помогло, спасибо.
